L1Trigger/L1TMuonEndCap/src/PrimitiveConversion.cc

```diff
@@ -553,11 +575,35 @@ void PrimitiveConversion::convert_rpc_details(EMTFHit& conv_hit) const {
   int fph = conv_hit.Phi_fp();
   int th  = conv_hit.Theta_fp();
 
+  bool use_cppf_coords = true;
+#ifdef PHASE_TWO_TRIGGER
```

Comment:
Is this a proper way to configure Phase-I vs. Phase-II?  @rekovic would know better how this is being done elsewhere in the L1T emulator.

Reply:
There is no switch for Phase2, because there is no actual algorithm for Phase2 yet. It will be updated in the future. The codes related to Phase2 are hidden by using "#ifdef" at the moment until it's ready.